m3u8/parser.py

```diff
@@ -348,8 +358,19 @@ def _cueout_envivio(line, state, prevline):
     else:
         return None
 
-def _parse_cueout_start(line, state, prevline):
-    _cueout_state = _cueout_elemental(line, state, prevline) or _cueout_envivio(line, state, prevline)
+def _cueout_simple(line):
+    # this needs to be called after _cueout_elemental
+    # as it would capture those cues incompletely
+    param, value = line.split(':', 1)
+    res = re.match('^(\d+(?:\.\d)?\d*)$', value)
+    if res:
+        return (None, res.group(1))
+
+def _parse_cueout(line, state, prevline):
+    _cueout_state = (_cueout_no_duration(line)
```

Comment:
`any` function can be used here. Do you agree?

Reply:
I did consider this, but I don't think so. Because of the way `_cueout_state` is used in the conditional below - it either needs to falsey, or a tuple. `any` can only return a boolean, unless I am missing something?

Reply:
You are right!